packages/mdc-typography/_variables.scss

```diff
@@ -106,5 +106,12 @@ $mdc-typography-styles: (
     font-weight: map-get($mdc-typography-font-weight-values, regular),
     letter-spacing: .08em,
     margin: -.5rem 0 1rem -.04em /* -8sp 0 16sp -.04em */
+  ),
+  button: (
+    font-size: .875rem, /* 14sp */
```

Comment:
MDC Button actually uses 14px font-size and 36px line-height explicitly.

Are you changing this on purpose?